tests/imap_test/imap_test.lf

```diff
@@ -0,0 +1 @@
+this is a test fact that maps to an image
```

Comment:
It would be nice if the text were related to the test to make it easy to tell which test the output corresponds to.

Reply:
I mean that's why I made the text state that it should map to an image? Do you feel like there's a more descriptive lf we could use?